### PR TITLE
Fix runbook URLs test

### PR DESCRIPTION
**Pull request by assafad**
Covered in [#7843](https://github.com/kubevirt/kubevirt/pull/7843).

This PR fixes runbook_url tests and adds `runbook_url` annotations for `VirtApiRESTErrorsHigh` and `VirtApiRESTErrorsBurst`.

Signed-off-by: assafad <aadmi@redhat.com>

```release-note
None
```

### CODE REVIEW

**Comment by assafad** [comment]
Hi @enp0s3, can you please have a look?
CC @sradco

File: pkg/virt-operator/resource/generate/components/prometheus.go [hunk 1]
```diff
@@ -390,6 +391,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 						},
 						Labels: map[string]string{
 							severityAlertLabelKey: "critical",
+							"runbook_url":         runbookUrlBasePath + "VirtApiRESTErrorsBurst",
```
Comment (sradco) [inline pkg/virt-operator/resource/generate/components/prometheus.go:394]: @assafad the "runbook_url" should be under Annotations

Comment (assafad) [reply pkg/virt-operator/resource/generate/components/prometheus.go]: Thanks, done

**Pull request closed without merging**